istioctl/pkg/verifier/verifier.go

```diff
@@ -359,6 +378,33 @@ func (v *StatusVerifier) verifyPostInstall(visitor resource.Visitor, filename st
 	return crdCount, istioDeploymentCount, err
 }
 
+func (v *StatusVerifier) getProfile(iop *v1alpha1.IstioOperator) string {
+	profile := "default"
+	if iop != nil && iop.Spec != nil && iop.Spec.Profile != "" {
+		profile = iop.Spec.Profile
+	}
+	return profile
+}
+
+func (v *StatusVerifier) getMergedIOP(userIOPStr, profile string) (*v1alpha1.IstioOperator, error) {
```

Comment:
i feel like this type of function is fairly core to the operator libs and should be exported rather than copied.

Reply:
Resolved.